_targets.R

```diff
@@ -0,0 +1,150 @@
+library(targets)
+library(tarchetypes)
+library(reticulate)
+
+yaml_file = 'test_config.yml'
+
+# MUST READ ---------------------------------------------------------------
+
+# IMPORTANT NOTE:
+#
+# you must execute the command 'earthengine authenticate' in a zsh terminal 
+# before initializing this workflow. See the repository README for complete 
+# dependencies and troubleshooting.
+
+# RUNNING {TARGETS}:
+#
+# run tar_make() to run the pipeline
+# run tar_visnetwork() to see visual summary
+
+
+# Set up python virtual environment ---------------------------------------
+
+if(!dir.exists('env')) {
+  source('data_acquisition/src/pySetup.R')
+} else {
+  use_condaenv(file.path(getwd(), 'env'))
+}
+
+
+# Source functions --------------------------------------------------------
+
+source('data_acquisition/src/general_functions.R')
+source_python('data_acquisition/src/gee_functions.py')
+
+# Define {targets} workflow -----------------------------------------------
+
+# Set target-specific options such as packages.
+tar_option_set(packages = "tidyverse")
+
+# target objects in workflow
+list(
+  tar_target(name = checkYmlMod,
+             command = return(file.info(yaml_file)$mtime),
+             cue = tar_cue('always')), 
+  
+  tar_target(name = readYmlMod,
+             command = tar_read(checkYmlMod),
+             cue = tar_cue('always')),
+  
+  # load, format, save yml as a csv
+  tar_target(name = ymlSave,
+             command = {
+               readYmlMod
+               formatYaml(yaml_file)
+               },
+             packages = c('yaml', 'readr')),
+  
+  # track the yml file
+  tar_file_read(name = yml,
+                command = {
+                  readYmlMod
+                  tar_read(ymlSave)
+                  },
+                read = read_csv(!!.x),
+                packages = 'readr'),
+  
+  # load, format, save locs as an updated csv
+  tar_target(name = locsSave,
+             command = {
+               readYmlMod
+               grabLocs(yml)
+               },
+             packages = 'readr'),
+
+  # track locs file
+  tar_file_read(name = locs,
+                command = {
+                  readYmlMod
+                  tar_read(locsSave)
```

Comment:
Do you need to include `tar_read()` here? I wouldn't think so, but perhaps with what you're using doing with target tracking you do?

Reply:
Nope! That was a newbie targets issue. Fixed that in this instance and the one above it.